lib/interferon/destinations/datadog.rb

```diff
@@ -42,6 +44,8 @@ def existing_alerts
         resp = @dog.get_all_alerts()
         alerts = resp[1]['alerts']
 
+        raise 'Failed to retrieve current alerts from datadog' if alerts.nil?
```

Comment:
we should probably print more debug information than that -- what was the cause of the error? something like this would be appropriate: https://github.com/airbnb/interferon/blob/9ad942b96c410d47710966c3006ed5a9d007546c/lib/interferon/destinations/datadog.rb#L112-L116. might be time to factor that out into a function?